### PR TITLE
Merging feature/model_api_4

### PR DESCRIPTION
**Pull request by HermanPlay**

### CODE REVIEW

File: containerized_ml_models/model_api_4/api.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    return read_model(arr)
+
+
+def read_model(data):
+    with open("model.pkl", "rb") as model_file:
+        model = pickle.load(model_file)
+    return model.predict(data).tolist()
+
+
+app.run(port=8000, host="0.0.0.0")
```
Comment (AdamixD) [inline containerized_ml_models/model_api_4/api.py:26]: In my opinion, you should add:

if __name__ == "__main__":
     app.run(host="0.0.0.0")

Comment (AdamixD) [inline containerized_ml_models/model_api_4/api.py:26]: In my opinion, you should add:

if name == "main":
app.run(host="0.0.0.0")

Comment (AdamixD) [inline containerized_ml_models/model_api_4/api.py:26]: In my opinion, you should add:

if name == "main":
app.run(host="0.0.0.0")

Comment (HermanPlay) [reply containerized_ml_models/model_api_4/api.py]: Done

**Review by matined — APPROVED** [review]

**Pull request merged**